docs/auth/index.md

```diff
@@ -403,3 +403,45 @@ providerFactories: {
   ghe: providers.github.create(),
 },
 ```
+
+## Configuring token issuers
+
+By default, Backstage uses the `TokenFactory` for signing the `Authorization` header. The `TokenFactory` generates a
+new private key for every instance every 30 minutes. Public keys are shared between instances via the database backend.
```

Comment:
As a reader of this doc you generally don't really care about how the auth backend itself implements the key management or what the internal interfaces are called. I think it's best to reword this a bit to the from a more user point of view, i.e. the auth backend generates and manages its own signing keys automatically for issued Backstage Tokens

Reply:
I've reworded the documentation, is this better? I'm open to suggestions!